content/en/docs/concepts/storage/volumes.md

```diff
@@ -462,92 +462,155 @@ and then removed entirely in the v1.26 release.
 
 ### hostPath {#hostpath}
 
-{{< warning >}}
-HostPath volumes present many security risks, and it is a best practice to avoid the use of
-HostPaths when possible. When a HostPath volume must be used, it should be scoped to only the
-required file or directory, and mounted as ReadOnly.
-
-If restricting HostPath access to specific directories through AdmissionPolicy, `volumeMounts` MUST
-be required to use `readOnly` mounts for the policy to be effective.
-{{< /warning >}}
-
 A `hostPath` volume mounts a file or directory from the host node's filesystem
 into your Pod. This is not something that most Pods will need, but it offers a
 powerful escape hatch for some applications.
 
+{{< warning >}}
+Using the `hostPath` volume type presents many security risks.
+If you can avoid using a `hostPath` volume, you should. For example,
+define a [`local` PersistentVolume](#local), and use that instead.
+
+If you are restricting access to specific directories on the node using
+admission-time validation, that restriction is only effective when you
+additionally require that any mounts of that `hostPath` volume are
+**read only**. If you allow a read-write mount of any host path by an
+untrusted Pod, the containers in that Pod may be able to subvert the
+read-write host mount.
+
+---
+
+Take care when using `hostPath` volumes, whether these are mounted as read-only
+or as read-write, because:
+
+* Access to the host filesystem can expose privileged system credentials (such as for the Kubelet) or privileged APIs
+  (such as the container runtime socket), that can be used for container escape or to attack other
+  parts of the cluster.
+* Pods with identical configuration (such as created from a PodTemplate) may
+  behave differently on different nodes due to different files on the nodes.
+{{< /warning >}}
+
 For example, some uses for a `hostPath` are:
 
-* running a container that needs access to Docker internals; use a `hostPath`
-  of `/var/lib/docker`
-* running cAdvisor in a container; use a `hostPath` of `/sys`
-* allowing a Pod to specify whether a given `hostPath` should exist prior to the
-  Pod running, whether it should be created, and what it should exist as
+* running a container that needs access to node-level system components
+  (such as reading system logs using a read-only mount of `/var/log`)
```

Comment:
```suggestion
  such as reading system logs using a read-only mount of `/var/log`
```

Reply:
Making the change proposed here could make readers backtrack. I'll reword, but the parens are needed.

Reply:
I cannot recall where I read it ... but the general guideline for technical writing is to avoid parenthesis when possible. We lost nothing by removing them.

Reply:
We do lose something. It otherwise becomes a [garden path sentence](https://en.wikipedia.org/wiki/Garden-path_sentence) where someone reading in English initially thinks that “reading system logs” qualifies which kind of node-level system components the container needs access to.

I'll take a rewritten phrase, sure, but this suggestion introduces something a bit like a code regression.